cluster/common.sh

```diff
@@ -933,3 +933,19 @@ function parse-master-env() {
   KUBELET_CERT_BASE64=$(get-env-val "${master_env}" "KUBELET_CERT")
   KUBELET_KEY_BASE64=$(get-env-val "${master_env}" "KUBELET_KEY")
 }
+
+# Writes the number of master replicas ($1) to the cluster config map.
+function update-cluster-config-map() {
+  local num_replicas="${1}"
+  echo "Updating cluster config map, the new number of replicas: ${num_replicas}"
+  local kubectl="${KUBE_ROOT}/cluster/kubectl.sh"
+  "${kubectl}" replace --force -f - <<EOF
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: apiserver-config
```

Comment:
Just a random thought, how does that play with federation? Would it be better to call the configmap `cluster-<some-cluster-identifier>`? I'm not aware of the differences when working with configmaps (and other API objects) in federated clusters so I'm not sure if this would be worth considering or not.

In general it's a good idea to do `associated-component.purpose-of-value` for the keys in the configmap, which will help you extend it if (assuming you go with `cluster-config`) you want other components to have config payloads in the `cluster-config` configmap in the future, and also makes it clear what each thing in the configmap is for.